### PR TITLE
Milestone 1: create toolbar and headline section

### PR DESCRIPTION
**Pull request by estebanmual**
### Setting up and starting mobile version skeleton

In this branch I:

- Added the Linters to the project
- Created different files that the project will have, such as: 
   - index.html
   - home.html
   - styles.css
- Added and completed the toolbar
-  Added and completed the headline section

### CODE REVIEW

**Review by ignatius22 — CHANGES_REQUESTED** [review]
Hi @Estebanmual , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

- [x] The container of the section is a `header` at least no header has been used, kindly implement this.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+       <header>
+            <nav class="nav-wrapper">
+            
+                <a href="#" class="link-hamburger-icon">☰</a>
+                <!-- <ul>
+                    <li><a href="#">Hello</a></li>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul> -->
```
Comment (Lordkaito) [inline index.html:23]: - [ ] [OPTIONAL] Kindly remove unused lines of code to keep your codebase cleaner.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            </nav> 
+
+
```
Comment (Lordkaito) [inline index.html:26]: - [ ] [OPTIONAL] Please, make sure to avoid empty lines, white spaces, and unused or commented code.

**Review by Lordkaito — APPROVED** [review]
Hi @Estebanmual , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**